### ISSUE TITLE
"gem install heckscher" not available

### ISSUE THREAD

**Issue by samfrench** [stevenjack/heckscher#6]
## Problem

The heckscher gem does not seem to be available in RubyGems.

I can clone the repo instead of installing the gem, but the docs indicate it would be in RubyGems.

**Comment by stevenjack** [comment]
@samfrench me and @kenoir created this whilst we were up for one of the elections, it's certainly not finished :) I think that's the default boilerplate readme that bundler creates when you make a new gem

**Comment by samfrench** [comment]
Fair enough. I see this relies on one of @kenoir's gems. The way around it is to clone the needed repos, then run `rake build` in each one, to install the gems locally.

Alternatively, I could point a Gemfile to the repos needed, to get them that way.

I am looking to do a delete of all items in DynamoDB. There was an endpoint previously which did that, but I assume that's not scripted anywhere? I was going to take this as a start and work from there. It looks like I need to paginate as there's not an easy way to delete all items.

**Comment by stevenjack** [comment]
@samfrench which gem of @kenoir does it rely on? I don't think the endpoint was ever removed... I can't remember the name of the app it was on though..

Is it a one of or something you want to be able to use a number of times?

**Comment by samfrench** [comment]
I thought the gem this relied on was the "dynamodb-exporter":

> https://github.com/stevenjack/heckscher/blob/ea3bb539dd3625a399a4b3a4cbb0e6bc85aa9b02/Rakefile#L3

I can see this under Kenny:

> https://github.com/kenoir/dynamodb-exporter

But then that is forked from this repo anyway...

For now, we want to delete items daily, before rehearsals. The job would be on the old Jenkins, so there's not much to go on.

I'll take a look at the aws docs, and paginate over the items to delete them. I think that is what you must have done previously.

**Comment by samfrench** [comment]
I wrote a quick script to loop over the items in Dynamo and delete them.

I also found out our throttling alarms are working, so that was a good thing from doing this.

**Comment by kenoir** [comment]
One annoying solution is simply to delete the tables and recreate them but
that probably doesn't play nice with your setup.

**Comment by samfrench** [comment]
Thanks @kenoir, I saw that solution when I researched. I think that would probably recreate the tables with different ids appended, so we would have to recreate the broker/renderer roles to apply to the tables, and also adjust the broker/renderer config to pick up the new arns. I could be wrong, but it seems likely.

**Comment by kenoir** [comment]
Yup, definitely create new ids, so unless you've got some method of
discovering the resources automatically it'll be a pain.
